compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/BasicContextMenuRepresentation.desktop.kt

```diff
@@ -84,7 +86,15 @@ class DefaultContextMenuRepresentation(
             Popup(
                 focusable = true,
                 onDismissRequest = { state.status = ContextMenuState.Status.Closed },
-                popupPositionProvider = rememberCursorPositionProvider()
+                popupPositionProvider = rememberCursorPositionProvider(),
+                onKeyEvent = {
```

Comment:
Can/should user add custom Esc handling?

Reply:
Probably we can introduce onKeyEvent parameter for dropdown menu, context menu, and alert dialog in the future, but that is a more complex change, and we need to think about it.

Reply:
right now, users can implement their own ContextMenuRepresentation, AlertDialogProvider, or pass focusable=false to dropdown menu (to disable Escape reacting)